src/pages/CategoriesPage.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCategoryTitle } from "../util/spotify";
+import { getCategoryPlaylists } from "../util/spotify";
+import CategoriesView from "../components/CategoriesView";
+
+export default function CategoriesPage(props) {
+  const { id } = useParams();
+  const [CategoryType, setCategoryType] = useState();
```

Comment:
`CategoryType` and `CategoryPlaylists` should start with lowercase letters as per project style